cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/NativeUtils.java

```diff
@@ -0,0 +1,344 @@
+
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        List<ReflectConfigClass> classList = new ArrayList<>();
+
+        int tally = 1;
+
+        for (Module module : currentPackage.modules()) {
+            String name = module.moduleName().toString();
+
+            //  {
+            //      "name":"<org>.<name>$test.0.<class>",
+            //      "methods":[
+            //          {"name":"$moduleInit","parameterTypes":["io.ballerina.runtime.internal.scheduling.Strand"] },
+            //          {"name":"$moduleStart","parameterTypes":["io.ballerina.runtime.internal.scheduling.Strand"] },
+            //          {"name":"$moduleStop","parameterTypes":["io.ballerina.runtime.internal.scheduling
+            //          .RuntimeRegistry"] }
+            //      ]
+            //  }
+
+            ReflectConfigClass testInitClass = new ReflectConfigClass(getQualifiedClassName(org, name, version,
+                    MODULE_INIT_CLASS_NAME));
+
+            testInitClass.addReflectConfigClassMethod(
+                    new ReflectConfigClassMethod(
+                            "$moduleInit",
+                            new String[]{"io.ballerina.runtime.internal.scheduling.Strand"}
+                    )
+            );
+
+            testInitClass.addReflectConfigClassMethod(
+                    new ReflectConfigClassMethod(
+                            "$moduleStart",
+                            new String[]{"io.ballerina.runtime.internal.scheduling.Strand"}
+                    )
+            );
+
+            testInitClass.addReflectConfigClassMethod(
+                    new ReflectConfigClassMethod(
+                            "$moduleStop",
+                            new String[]{"io.ballerina.runtime.internal.scheduling.RuntimeRegistry"}
+                    )
+            );
+
+            //  {
+            //      "name":"<org>.<name>$test.0.$configurationMapper",
+            //      "methods":[{"name":"$configureInit","parameterTypes":["java.lang.String[]","java.nio.file
+            //      .Path[]","java.lang.String"] }]
+            //  }
+
+            ReflectConfigClass testConfigurationMapper = new ReflectConfigClass(getQualifiedClassName(org, name,
+                    version, MODULE_CONFIGURATION_MAPPER));
+
+            testConfigurationMapper.addReflectConfigClassMethod(
+                    new ReflectConfigClassMethod(
+                            "$configureInit",
+                            new String[]{"java.lang.String[]", "java.nio.file.Path[]", "java.lang.String"}
+                    )
+            );
+
+            // {
+            //      "name":"<org>.<name>$test.0.test_execute-generated_1",
+            //      "methods":[{"name":"__execute__","parameterTypes":["io.ballerina.runtime.internal.scheduling
+            //      .Strand",
+            //      "io.ballerina.runtime.api.values.BString","boolean","io.ballerina.runtime.api.values.BString",
```

Comment:
Do we need these comments here? The Strings starting from L129 explains it.

Reply:
Removed all config comments.